### DISCUSSION TITLE
Welcome, Moises Reyes!

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#270] [Subscribe]
Ahoy @MeeL1ght, welcome aboard!

Thank you for joining us! Every day there are new quiz questions posted in [Discussions](https://github.com/ParanoidUser/yolo/discussions/categories/galaxy-trivia), you can try to answer them to earn the Galaxy Brain badge. Spread the word, collaborate and have fun, have fun, have fun! 😊

Let me know if you have any questions.

**Comment by ghost** [comment]
Hello! @ParanoidUser Thanks a lot! I think it is excellent. I will gladly contribute as soon as possible 😎

**Reply by ParanoidUser** [reply]
Thank you for your active contribution!
I know that [Pair Extortionate](https://github.com/MeeL1ght?tab=achievements&achievement=pair-extraordinaire) can be a little tricky because you have to list other co-authors (besides yourself). Congratulations on new badges! 😊

**Reply by ghost** [reply]
Ty! ;D
